src/Infrastructure/Manifest/Normalizer/PhpManifestNormalizer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Manifest\Normalizer;
+
+use InvalidArgumentException;
+use SprykerSdk\Sdk\Core\Application\Dependency\SettingFetcherInterface;
+use SprykerSdk\SdkContracts\Enum\Setting;
+use Symfony\Component\Process\Process;
+
+class PhpManifestNormalizer implements ManifestNormalizerInterface
+{
+    /**
+     * @var int
+     */
+    protected const PHPCBF_SUCCESS_FIX_CODE = 1;
+
+    /**
+     * @var \SprykerSdk\Sdk\Core\Application\Dependency\SettingFetcherInterface
+     */
+    protected SettingFetcherInterface $settingFetcher;
+
+    /**
+     * @param \SprykerSdk\Sdk\Core\Application\Dependency\SettingFetcherInterface $settingFetcher
+     */
+    public function __construct(SettingFetcherInterface $settingFetcher)
+    {
+        $this->settingFetcher = $settingFetcher;
+    }
+
+    /**
+     * @param string $filePath
+     *
+     * @throws \InvalidArgumentException
+     *
+     * @return void
+     */
+    public function normalize(string $filePath): void
+    {
+        $sdkDir = $this->settingFetcher->getOneByPath(Setting::PATH_SDK_DIR)->getValues();
+
+        $process = Process::fromShellCommandline('php "${:sdk_path}"/vendor/bin/phpcbf "${:filepath}"', $sdkDir);
```

Comment:
why this command depends on phpcbf ?

Reply:
This is php file normalizer for fixing cs issues on generated php files. Because we assemble files by the parts in twig and we unable to provide all the class imports and sort the imports. Also in case of updating cs rules we don't need to update all the files templates. The same way is use in spryk package.